### PR TITLE
adding Favorite stay from the product page

### PR DESCRIPTION
**Pull request by Konzisam**
image

adding Favorite stay from the product page #27

### CODE REVIEW

File: src/components/FavouriteStay/FavoriteStay.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    height: 32px;
+   
+
+}
+
+.host-badge-text {
+    padding-right: 4px 4px;
+    font-weight: 500;
+    font-size: 1.4rem; 
+    font-family: 'Montserrat', sans-serif;
```
Comment (dvnishshanka) [inline src/components/FavouriteStay/FavoriteStay.css:37]: I think you can remove this line and let the font family to inherit from its parent component. When I remove it it looks like this.
image

File: src/App.jsx [hunk 2]
```diff
@@ -22,6 +23,7 @@ function App() {
           <img src={reactLogo} className="logo react" alt="React logo" />
         </a>
       </div>
+      <FavoriteStay />
```
Comment (dvnishshanka) [inline src/App.jsx:26]: This component can be added the product page itself as it is already implemented.

File: src/components/FavouriteStay/FavoriteStay.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+          <div className="host-badge-text">Guest{<br></br>}favorite</div>
+
+          <div className="host-badge-right">
+          <img className="right" src={Right}
+            alt="fav-badge-right" />
+          </div>
+        </div>
+
+        <div></div>
```
Comment (dvnishshanka) [inline src/components/FavouriteStay/FavoriteStay.jsx:28]: This <div> tag seems redundant as there's nothing between the tags. There are few similar tags in this file.

File: src/components/StarRating/StarRating.scss
Comment (dvnishshanka) [inline src/components/StarRating/StarRating.scss:1]: You can remove this file as it is empty.

**Comment by dvnishshanka** [comment]
Also the as per the ticket mid text summary is shown.

File: src/components/FavouriteStay/FavoriteStay.css (hunk 1 above)
Comment (Konzisam) [reply src/components/FavouriteStay/FavoriteStay.css:37]: I adjusted the image and the requirements to match the updated actual website look, which is different from what is on the planning board.

File: src/components/StarRating/StarRating.scss
Comment (Konzisam) [reply src/components/StarRating/StarRating.scss:1]: thanks

File: src/components/FavouriteStay/FavoriteStay.jsx (hunk 3 above)
Comment (Konzisam) [reply src/components/FavouriteStay/FavoriteStay.jsx:28]: they are actually useful, amusing them in the grid-template-columns, probably there is a better way to do it

File: src/App.jsx (hunk 2 above)
Comment (Konzisam) [reply src/App.jsx:26]: thanks, changed that

**Comment by Konzisam** [comment]
I adjusted the image and the requirements to match the updated actual website look, which is different from what is on the planning board.

**Review by dvnishshanka — APPROVED** [review]
I Approved the PR. But it has some merge conflicts.

**Pull request merged**